modules/nw-ovn-kubernetes-features.adoc

```diff
@@ -5,17 +5,19 @@
 [id="nw-ovn-kubernetes-purpose_{context}"]
 = OVN-Kubernetes purpose
 
-The OVN-Kubernetes network plugin is an open-source, fully-featured Kubernetes CNI plugin that uses Open Virtual Network (OVN) to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution. The OVN-Kubernetes network plugin:
+The OVN-Kubernetes network plugin is an open-source, fully-featured Kubernetes CNI plugin that uses Open Virtual Network (OVN) to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution. The OVN-Kubernetes network plugin uses the following technologies:
 
-// OVN (Open Virtual Network) is consistent with upstream usage.
+* OVN to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution.
```

Comment:
The second sentence of this bullet is redundant IMO, as it uses the exact same phrasing used in the introduction.

Reply:
:heavy_check_mark: